dbhydra/dbhydra_core.py

```diff
@@ -729,20 +730,33 @@ def __init__(self, db1, name, columns=None, types=None):
         self.types = types
         print("==========================================")
 
+    def initialize_columns(self):
+        information_schema_table = Table(self.db1, 'INFORMATION_SCHEMA.COLUMNS')
+        query = f"SELECT COLUMN_NAME FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME  = '" + self.name + "';"
+        columns = information_schema_table.select(query)
+        self.columns = columns
+
+    def initialize_types(self):
+        information_schema_table = Table(self.db1, 'INFORMATION_SCHEMA.COLUMNS', ['DATA_TYPE'], ['nvarchar(50)'])
+        query = f"SELECT DATA_TYPE FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA TABLE_NAME  = '" + self.name + "'"
+        types = information_schema_table.select(query)
+        self.types = types
+
     def get_all_columns(self):
         information_schema_table = Table(self.db1, 'INFORMATION_SCHEMA.COLUMNS')
         query = "SELECT COLUMN_NAME FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME  = '" + self.name + "'"
         columns = information_schema_table.select(query)
-        print(columns)
+
         return (columns)
 
     def get_all_types(self):
+
         information_schema_table = Table(self.db1, 'INFORMATION_SCHEMA.COLUMNS', ['DATA_TYPE'], ['nvarchar(50)'])
-        query = "SELECT DATA_TYPE FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME  = '" + self.name + "'"
+        query = "SELECT DATA_TYPE FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA TABLE_NAME  = '" + self.name + "'"
```

Comment:
are you sure about "TABLE_SCHEMA TABLE_NAME" part?

Reply:
This was wrong, I had it fixed already, but forgot to commit. Anyways this was insufficient, because for example for varchars it didnt return the varchar size.

Reply:
fixed, more changes in next PR @DovaX